### PR TITLE
tls: add test for tls keywords v1

### PR DESCRIPTION
**Pull request by hsadia538**
Description:
Add tests for tls keywords: `tls.subject ` `tls.issuerdn`

### CODE REVIEW

File: tests/tls-issuerdn/test.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+requires:
```
Comment (jufajardini) [inline tests/tls-issuerdn/test.yaml:1]: I recently learned that when we want to use a pcap from a pre-existing sv test, we can also add:
```yaml
pcap: ../existing-test-name/pcap-file.pcap
```
:)

Comment (jufajardini) [reply tests/tls-issuerdn/test.yaml:1]: As seen in: https://github.com/OISF/suricata-verify/blob/2eca390bdd825fd7c709a18ef0884146c3cb4c7e/tests/detect-udp-flow-rule-01/test.yaml

File: tests/tls-issuerdn/test.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: tls
+        tls.session_resumed: true
+
```
Comment (jufajardini) [inline tests/tls-issuerdn/test.yaml:11]: Suggestion: let's `tls.issuerdn` here, as well, as this also helps to showcase how the output is the same, regardless of using the legacy keyword or the more recent one. ;)

File: tests/tls-subject/test.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: tls
+        tls.session_resumed: true
```
Comment (jufajardini) [inline tests/tls-subject/test.yaml:10]: Similar idea as above: let's check for `tls.suject` here :)

**Review by jufajardini — CHANGES_REQUESTED** [review]
We're on the right path, good job!

Some inline comments, but just to improve on the work you've done.
Partly related: while reviewing this PR I noticed that at least the current `tls` keywords for these legacy keywords are not covered by suricata-verify tests. Could you please also add tests for those, and check if we are missing any other in the process?
The ones I checked were: `tls.cert_issuer` and `tls.cert_subject`.

The commit messages need two minor fixes:
- they are missing the blank line between the commit subject and the commit body. (if you open the branch in tig, you'll notice that the `Ticket` part is appended to the commit title, due to the lack of that extra line). 
- let's also go back to the `Ticket #xxx` format, as this is more according to our guidelines :)

**Comment by hsadia538** [comment]
> We're on the right path, good job!
> 
> Some inline comments, but just to improve on the work you've done. Partly related: while reviewing this PR I noticed that at least the current `tls` keywords for these legacy keywords are not covered by suricata-verify tests. Could you please also add tests for those, and check if we are missing any other in the process? The ones I checked were: `tls.cert_issuer` and `tls.cert_subject`.
> 
Do you want me to add the tests for those in the same commits?

File: tests/tls-issuerdn/test.yaml (hunk 1 above)
Comment (hsadia538) [reply tests/tls-issuerdn/test.yaml:1]: so you mean I dont need to give the pcap at all and reference it here ?

Comment (hsadia538) [reply tests/tls-issuerdn/test.yaml:1]: Yes it works. thanks

**Comment by hsadia538** [comment]
Followed by #1075

**Pull request closed without merging**